### PR TITLE
Merged Discord Templates and discord.bio into discords.com.

### PR DESCRIPTION
**Pull request by DiscordAnaxes**
Why? 

Discord.bio & Discord Templates have been acquired by discords.com, and they have recently moved discord.bio and discord templates to their website.

Evidence:

<details>
  <summary>Discord.bio</summary>

</details>

<details>
  <summary>Discord Templates</summary>

</details>

### CODE REVIEW

File: websites/D/Discords.com/presence.ts [hunk 1]
```diff
@@ -195,14 +195,136 @@ presence.on("UpdateData", async () => {
...
     ];
+    // discord.bio
+  } else if (document.location.pathname.includes("/profiles")) {
+    presenceData.largeImageKey = "discordbio_logo";
+    presenceData.smallImageKey = "reading";
+    presenceData.smallImageText = "Browsing...";
+    presenceData.details = "Browsing top bios";
+    presenceData.buttons = [
+      {
+        label: "View Bios",
```
Comment (ririxidev) [inline websites/D/Discords.com/presence.ts:206]: i don't think it's correct

**Review by rxri — DISMISSED** [review]
i think it's ok but only this string

File: websites/D/Discords.com/presence.ts (hunk 1 above)
Comment (DiscordAnaxes) [reply websites/D/Discords.com/presence.ts:206]: In what terms? Code or like grammatically?

Comment (ririxidev) [reply websites/D/Discords.com/presence.ts:206]: grammatically

Comment (DiscordAnaxes) [reply websites/D/Discords.com/presence.ts:206]: I was confused with that at the start too, 'cause I had bio's.

Comment (ririxidev) [reply websites/D/Discords.com/presence.ts:206]: english is weird

**Review by rxri — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Comment by ririxidev** [comment]
good pr 👌

**Pull request merged**